smart-contracts-tutorial/first-contract.md

```diff
@@ -109,30 +101,53 @@ make scenario3-run-client
 
 Open a web browser to http://localhost:8000/
 
-<small>
+```
 currently, this is the UI of the Pixel Gallery demo
 TODO : provide a proper UI
-</small>
+```
 
 In the REPL, try the following commands in the same order:
 ```
 home
-home!game!guess(14)
-home!game!guess(37)
+host = home~.contractHost
+installation = host~.install({start: home.guess37ContractSource})
+invites = installation~.spawn()
+playerInvite = invites~.playerInvite
+seat = host~.redeem(playerInvite)
+
+seat~.guess(14)
+seat~.guess(37)
 ```
 
-The first `home` command shows what was defined in `lib/ag-solo/vats/vat-demo.js` as `chainBundle`
+### Commands breakdown 
+
+The first `home` command shows what was defined in `lib/ag-solo/vats/vat-pixel.js` as return value of `createPixelBundle`
+It is a presence for an object containing the properties `contractHost`, `handoffService` and `guess37ContractSource`
 
-`home!game!guess(14)` is a losing guess
-`home!game!guess(37)` is a winning guess
+`host = home.contractHost` retrieves the contractHost presence and stores it in the `host` variable for convenience
+
+`installation = host~.install({start: home.guess37ContractSource})` installs the guess37 contract in the contract host
+
+`invites = installation~.spawn()` runs the contract (here, the `guess37Contract` function) and store the invites bundle returned by the contract
+
+`playerInvite = invites~.playerInvite` extracts the `playerInvite` from the invites bundle
+
+`seat = host~.redeem(playerInvite)` redeems the invitation. This allows to participate to the contract. What is returned is stored in a `seat` variable as if the contract/game was a table and you were retrieving a seat at this table
```

Comment:
"This allows you to participate in the contract." 

"Redeem returns an object that represents a right to participate in the contract (a seat at the table)."

A seat at the table is a common enough metaphor that it doesn't deserve an explanation here.